__tests__/app.test.js

```diff
@@ -426,4 +428,24 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
     });
   });
-});
\ No newline at end of file
+});
+
+describe("ERRORS", () => {
```

Comment:
these errors are essentially doing the same thing - they're testing non-existent routes in your app - the endpoints you are testing here actually have nothing to do with your users endpoint as they don't match it so we'll never reach it - it's the same as requesting /api/bananas! :D 
So we only really need a single test for a non-existent route, and test one not related to your other endpoints so it's less confusing what you are testing!